pelicanconf.py

```diff
@@ -1,3 +1,29 @@
-# -*- coding: utf-8 -*-
+#!/usr/bin/env python
+# -*- coding: utf-8 -*- #
+from __future__ import unicode_literals
```

Comment:
We stick with Python 3.5 so this line is also unnecessary (it's just way to make unicode literals available also in Python 2).